package.json

```diff
@@ -17,6 +17,7 @@
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.5",
     "flatbuffers": "^1.12.0",
+    "moment": "^2.27.0",
```

Comment:
momentjs [is end of life](https://momentjs.com/docs/#/-project-status/) and sometimes criticized for being huge; consider dayjs, which has a [similar fromNow API](https://day.js.org/docs/en/display/from-now)

Reply:
Ooof, when I started doing JS moment was the new hotness...I'll migrate to this new world